examples/cloudtrail/cloudtrail.tf

```diff
@@ -5,10 +5,10 @@ module "cloudtrail" {
   source = "coralogix/aws/coralogix//modules/cloudtrail"
 
   coralogix_region   = "Europe"
-  CustomDomain       = "https://<your custom doamin>/api/v1/logs"
+  custom_domain      = "https://<your custom doamin>/api/v1/logs"
```

Comment:
If you have coralogix_region configured no need for custom_domain and vise-versa